packages/react/src/filter-sidebar/FilterSidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { PropsWithChildren } from 'react';
+import { FormStack } from '@ag.ds-next/react/form-stack';
+import { CollapsingSideBar } from '../_collapsing-side-bar';
+
+export type FilterSidebarProps = PropsWithChildren<{
+	/** The aria-label for the FilterSidebar. */
+	'aria-label'?: string;
```

Comment:
I find it a little odd that we can customise the aria-label but not the title. 

- I feel like `title` should be a prop (`"Filters"` by default) 
- Do we really need the aria-label prop?